runtime/builtin_types.go

```diff
@@ -541,6 +541,18 @@ func builtinRepr(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
 	return s.ToObject(), nil
 }
 
+func builtinSorted(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+	if raised := checkFunctionArgs(f, "sorted", args, ObjectType); raised != nil {
```

Comment:
I noticed that as well, but didn't explicitly mention it b/c @S-YOU put `Add builtin sorted, without cmp, key, and reverse options.` in the change request message.